system.go

```diff
@@ -545,3 +594,48 @@ func (av *AirVantage) ApplySettings(settings map[string]interface{}, delete []st
 	}
 	return string(res.Operation), nil
 }
+
+// SendCommand launch an operation to run the given command and parameters on the system
+func (av *AirVantage) SendCommand(commandID string, parameters map[string]interface{}, protocol, systemUID string) (string, error) {
+
+	type jsonBody struct {
+		Systems struct {
+			UIDs []string `json:"uids"`
+		} `json:"systems"`
+		CommandID  string                 `json:"commandId"`
+		Parameters map[string]interface{} `json:"parameters"`
+		Protocol   string                 `json:"protocol"`
+		Reboot     bool                   `json:"reboot"`
```

Comment:
reboot parameter in the command API?

Reply:
You are right, reboot is not part of the sendCommand parameters, I will remove it.